modules/rhcos-enable-node-scan-scannerv4.adoc

```diff
@@ -20,7 +20,7 @@ For information about supported platforms and architecture, see the link:https:/
 
 [NOTE]
 ====
-Node scanning with Scanner V4 is enabled by default in a new installation of release 4.8 and later. These steps are only required if you are updating from an earlier version of {product-title-short} and Scanner V4 was not enabled.
+Node scanning with Scanner V4 is enabled by default in a new installation of release 4.8 and later. These steps are only required if you are updating from an earlier version of {product-title-short} and Scanner V4 was not enabled. Although the StackRox Scanner is deprecated as of release 4.9, it still must be enabled on the cluster where Central is installed due to software dependencies. It can be safely disabled on secured clusters if Scanner V4 is enabled or if scanning by using secured clusters is not needed.
```

Comment:
Node scanning does not use scanner in the secured cluster - this last sentence may be unnecessary here: _"It can be safely disabled on secured clusters if Scanner V4 is enabled or if scanning by using secured clusters is not needed."_